meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -74,3 +74,4 @@ RESIN_FLASHER_FLAG_FILE = "resin-image-flasher"
 INITRAMFS_IMAGE = "resin-image-initramfs"
 INITRAMFS_IMAGE_BUNDLE = "1"
 KERNEL_INITRAMFS = "-initramfs"
+INITRAMFS_TASK = ""
```

Comment:
@floion this is introduced by the tegra BSP (because they feel they need to define this), but it should be defined as we make use of the proper mechanism with INITRAMFS_IMAGE.